gravitee-apim-portal-webui-next/src/app/api-details/api-tab-subscriptions/api-tab-subscriptions.component.ts

```diff
@@ -33,17 +37,28 @@ export class ApiTabSubscriptionsComponent implements OnInit {
   apiId!: string;
 
   subscriptionsList$!: Observable<SubscriptionData[]>;
+  subscriptionsMetadataList!: SubscriptionMetadata;
+  displayedColumns: string[] = ['application', 'plan', 'status', 'expand'];
   private subscriptionService = inject(SubscriptionService);
 
-  constructor() {}
+  constructor(private capitalizeFirstPipe: CapitalizeFirstPipe) {}
 
   ngOnInit(): void {
     this.subscriptionsList$ = this.loadSubscriptions$();
   }
 
+  retrieveMetadataName(plan: string) {
```

Comment:
It's a bit confusing that the parameter is named `plan` but you are using it for applications as well. Why not rename it to `id`?

Reply:
Done